packages/vite/src/node/idResolver.ts

```diff
@@ -40,7 +40,7 @@ export function createIdResolver(
       pluginContainer = await createEnvironmentPluginContainer(
         environment as Environment,
         [
-          aliasPlugin({ entries: config.resolve.alias }), // TODO: resolve.alias per environment?
+          aliasPlugin({ entries: config.resolve.alias }),
```

Comment:
This should be `environment.options.resolve.alias`, the config has the default values. We were thinking that we should deprecate these defaults on the `ResolvedConfig` and remove them later to avoid these issues.

Reply:
I kept this as a fallback and collecting the environment specific `resolve.alias` in the plugin, but if it's not needed, I can change this to pass it directly. Makes total sense as this part is creating a plugin container for the environment.

Reply:
Ah, I see the new alias plugin. I think we don't need it. We can use the option here and keep everything else as is.